### PR TITLE
feat: Add long term statistics

### PR DESCRIPTION
**Pull request by rsegers**
This PR adds [long term statistics](https://data.home-assistant.io/docs/statistics/) for the following sensors:
- EV Battery
- EV Range
- Total Range
- Odometer

Other (non-EV) sensors might be interesting as well to add as long term statistics in HA, but I am only able to test EV-sensors.

### CODE REVIEW

**Comment by cdnninja** [comment]
Could we re-use the existing entity instead of two classes for this?

**Comment by rsegers** [comment]
Multiple inheritance for ```InstrumentSensor``` also works. Hadn't thought of that. Updated PR.

**Review by fuatakgun — CHANGES_REQUESTED** [review]
i forgot to publish my comments :)

File: custom_components/kia_uvo/sensor.py [hunk 1]
```diff
@@ -265,12 +286,13 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
...
             "mdi:update",
             DEVICE_CLASS_TIMESTAMP,
+            None,
         )
     )
     async_add_entities(sensors, True)
 
 
-class InstrumentSensor(KiaUvoEntity):
+class InstrumentSensor(KiaUvoSensorEntity):
```
[the remark this answers is not in the document]
Comment (rsegers) [reply custom_components/kia_uvo/sensor.py]: I think our comments just crossed each other :) I removed the ```KiaUvoSensorEntity``` class in favor of inheriting ```SensorEntity``` at ````InstrumentSensor````

[the remark this answers is not in the document]
Comment (fuatakgun) [reply custom_components/kia_uvo/sensor.py]: i am not good at using GitHub efficiently yet :-) looks good, thanks

**Comment by fuatakgun** [comment]
LGTM, @cdnninja please also check too.

**Pull request merged**